controllers/shooter.py

```diff
@@ -147,6 +135,7 @@ def find_allowable_angle(self, dist: float) -> float:
         Currently does not consider angle from target
         dist: planar distance from the target
         """
+        dist = min(dist, 14.0)
```

Comment:
Should we consider clamping with a max as well?

Reply:
Or would be 0 surely? Not a bad idea.